src/components/Typography/Typography.less

```diff
@@ -0,0 +1,54 @@
+@import (reference) '../../styles/variables.less';
+@import (reference) '../../styles/mixins.less';
+
+.@{prefix}-Typography {
+
+	&-h1,
+	&-h2,
+	&-h3 {
+		margin: 0;
+		font-weight: @font-weight-medium;
+		text-rendering: optimizelegibility;
+	}
+
+	// These values have been adjusted for the Xandr branding. Anything after h3
+	// shouldn't be used in practice.
+	&-h1 {
+		font-size: 24px;
+		line-height: 28px;
+	}
+	&-h2 {
+		font-size: 16px;
+		line-height: 20px;
+	}
+	&-h3 {
+		font-size: 12px;
+		line-height: 16px;
+		text-transform: uppercase;
+		color: @color-neutral-6;
+	}
+
+	&-a {
+		font-weight: @font-weight-medium;
+		color: @color-linkColor;
+		text-decoration: none;
+	}
+	&-a:hover {
+		color: @color-linkColorHover;
+		text-decoration: underline;
+	}
+
+	// NOTE: kbd and samp are currently not prop options for `variant`
```

Comment:
let's just remove these or add them as variants then?

Reply:
Seems super simple to just add them. I left them out because the JIRA ticket didn't mention them.